server/models/MatchModel.js

```diff
@@ -4,7 +4,10 @@ const mongoose = require('mongoose')
 const Schema = mongoose.Schema;
 
 const matchModel = new Schema({
-  userIDs: [String],
+  userIDs: [
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
+  ],
```

Comment:
array does not take key , value pairs. Array should have an object inside. Also there is a const already declared for mongoose.Schema, you can use the constant here `userIDs: [
    {
      type: Schema.Types.ObjectId,
      ref: 'User',
    },
  ],`
  Maybe something like this?

Reply:
Thanks for the reminder. It is modified.